traffic_ops/traffic_ops_golang/crconfig/handler.go

```diff
@@ -75,8 +77,17 @@ func SnapshotGetHandler(w http.ResponseWriter, r *http.Request) {
 		api.HandleErr(w, r, inf.Tx.Tx, http.StatusNotFound, errors.New("CDN not found"), nil)
 		return
 	}
-	w.Header().Set(rfc.ContentType, rfc.ApplicationJSON)
-	w.Write([]byte(`{"response":` + snapshot + `}`))
+
+	var decoded tc.CRConfig
+	if err = json.Unmarshal([]byte(snapshot), &decoded); err != nil {
+		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, fmt.Errorf("failed to unmarshal stored snapshot for cdn '%s': %v", inf.Params["cdn"], err))
+	}
+
+	if inf.Version.Major < 4 {
```

Comment:
Do we need to check `inf.Config.CRConfigEmulateOldPath` here as well?

Reply:
Yes, I think so. Unless we're removing that in this PR. So I'll wait to do anything until that's decided.

Reply:
@ocket8888 Since I think we can remove the flag in a separate GH issue, we might need the check here then.